m4/ltp-tirpc.m4

```diff
@@ -6,7 +6,11 @@ AC_DEFUN([LTP_CHECK_TIRPC],[
 	TIRPC_LIBS=""
 
 	AC_CHECK_HEADERS([tirpc/netconfig.h netconfig.h], [
-		TIRPC_CPPFLAGS="-I${SYSROOT}/usr/include/tirpc"
+		if [[ -n $SDKTARGETSYSROOT ]]; then
```

Comment:
`[[` and `[[` is not POSIX shell (it's a bashism). `if test -n "$SDKTARGETSYSROOT"; then` would be better.

From long term perspective, it might be good to introduce some config option `--with-sysroot=` which would allow to pass this config. But to be honest, I don't know much about this area.